includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -319,12 +321,18 @@ protected function get_row_data( $item ) {
 					$course_average_percent .= '%';
 				}
 
+				$days_to_completion = $this->get_course_days_to_completion( $item );
+				if ( null !== $days_to_completion ) {
+					$days_to_completion = Sensei_Utils::as_absolute_rounded_number( $days_to_completion, 2 );
+				}
+
 				$column_data = apply_filters(
 					'sensei_analysis_overview_column_data',
 					array(
-						'title'           => $course_title,
-						'completions'     => $course_completions,
-						'average_percent' => $course_average_percent,
+						'title'              => $course_title,
+						'completions'        => $course_completions,
+						'average_percent'    => $course_average_percent,
+						'days_to_completion' => $days_to_completion ?? __( 'n/a', 'sensei-lms' ),
```

Comment:
I think it would be best to make the letter casing consistent. We use uppercased N/A on some of the other reports.

@donnapep Do you have a preference? 🙂

Reply:
Oh, it looks like we introduced inconsistency with uppercase version :) 
https://github.com/Automattic/sensei/blob/712118ee9df336202c74839476c9c6ce04027699/includes/class-sensei-analysis-overview-list-table.php#L366